_includes/live-demos/autocompleter/index.js

```diff
@@ -13,31 +13,77 @@ tinymce.init({
   selector: 'textarea#autocompleter',
   height: 250,
   setup: function (editor) {
+    function onAction (autocompleteApi, rng, value) {
```

Comment:
Please avoid using named functions like this, as it's technically [invalid](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Paving_the_way_for_future_ECMAScript_versions) in strict mode (they are only allowed in the root context). It should be something like this instead:
```suggestion
    var onAction = function (autocompleteApi, rng, value) {
```